core/src/main/java/org/bitcoinj/evolution/SimplifiedMasternodeListDiff.java

```diff
@@ -88,29 +95,43 @@ protected void parse() throws ProtocolException {
         }
 
         size = (int)readVarInt();
-        mnList = new ArrayList<SimplifiedMasternodeListEntry>(size);
-        for(int i = 0; i < size; ++i)
-        {
+        mnList = new ArrayList<>(size);
+        for (int i = 0; i < size; ++i) {
             SimplifiedMasternodeListEntry mn = new SimplifiedMasternodeListEntry(params, payload, cursor, protocolVersion);
             cursor += mn.getMessageSize();
             mnList.add(mn);
         }
 
-        //0.14 format include quorum information
-        if(payload.length > cursor - offset) {
```

Comment:
This section is removed because we don't need to support Dash Core 0.13 protocols